src/code/z_eff_blure.c

```diff
@@ -726,10 +744,9 @@ void EffectBlure_SetupSimpleAlt(GraphicsContext* gfxCtx, EffectBlure* this, Vtx*
 void (*sSetupHandlers[])(GraphicsContext* gfxCtx, EffectBlure* this, Vtx* vtx) = {
     EffectBlure_SetupSimple,
     EffectBlure_SetupSimpleAlt,
+    NULL,
 };
```

Comment:
What would happen if you try to run a function with `NULL`? There's no safety checks to see if `drawMode` is less than 2, are you sure `NULL` should be in here?

Reply:
Agreed here. Probably better to keep the unused pad data.

Reply:
`sSetupHandlers` is only used on `EffectBlure_DrawSimple`, which has a check for `drawMode` being less than 2
https://github.com/zeldaret/mm/blob/b2535fc75a845adffbec760f58c2d654b99f2964/src/code/z_eff_blure.c#L1022-L1026

Reply:
Hmm, with the check it seems fine to actually move the NULL inside. I'm wondering if we should add a drawMode enum though?